app/lib/registries/organisations_registry.rb

```diff
@@ -30,7 +34,8 @@ def fetch_organisations_from_rummager
       params = {
         filter_format: 'organisation',
         fields: %w(title slug acronym),
-        count: 1500
+        count: 1500,
+        order: 'title'
```

Comment:
Clever, no need to sort later :)

Reply:
I have a slight reservation that Rummager shouldn't perhaps be used for this sort of thing - it should provide the data and then the apps should do filtering - but given the scale of data and the fact that Elasticsearch is optimised for this sort of thing, it makes sense.